src/core/channels/encrypted_channel.ts

```diff
@@ -46,14 +47,15 @@ export default class EncryptedChannel extends PrivateChannel {
     );
   }
 
-  /** Handles an event. For internal use only.
+  /** Handles an message. For internal use only.
    *
-   * @param {String} event
-   * @param {*} data
+   * @param {Message} message
    */
-  handleEvent(event: string, data: any) {
+  handleMessage(message: Message) {
+    var event = message.event;
+    var data = message.data;
     if (event.indexOf("pusher_internal:") === 0 || event.indexOf("pusher:") === 0) {
-      super.handleEvent(event, data);
+      super.handleMessage(message);
       return
     }
     this.handleEncryptedEvent(event, data)
```

Comment:
If `handleEvent` is being renamed and the signature changed, then I think this should be too.